.circleci/config.yml

```diff
@@ -284,6 +301,35 @@ jobs:
             echo "module.exports['compilers'] = {solc: {version: '$(realpath node_modules/solc/)'}}" > truffle-config.js
             node ../truffle/node_modules/.bin/truffle test
 
+  cli-smoke-test:
+    docker:
+      - image: circleci/node:17
+    steps:
+      - update-npm
+      - show-npm-version
+      - provision-and-package-solcjs
+      - run:
+          name: "CLI smoke test (repository)"
+          command: |
+            cd solc-js
+            dist/solc.js --version
+
+            echo "contract C {}" > C.sol
+            dist/solc.js C.sol --bin
+            [[ -f C_sol_C.bin ]]
+      - run:
+          name: "CLI smoke test (package)"
+          command: |
+            mkdir package/
+            cd package/
+            npm install ../solc-js.tgz
+
+            npx solcjs --version
+
+            echo "contract C {}" > C.sol
+            npx solcjs C.sol --bin
+            [[ -f C_sol_C.bin ]]
```

Comment:
This is a new test, not completely related to the PR but it's also a case that fails with the same error.

I could make this a separate PR but we should merge them together anyway.

Reply:
Fine to keep here - I actually thought about suggesting to add something simple like this :-).